packages/components/src/components/contract-card/contract-card-items/contract-card-sell.tsx

```diff
@@ -14,7 +14,10 @@ export type TContractCardSellProps = {
 
 const ContractCardSell = ({ contract_info, getCardLabels, is_sell_requested, onClickSell }: TContractCardSellProps) => {
     const is_valid_to_sell = isValidToSell(contract_info);
-    const should_show_sell = hasContractEntered(contract_info) && isOpen(contract_info);
+    const should_show_sell =
+        (hasContractEntered(contract_info) ||
+            isForwardStarting(contract_info?.shortcode ?? '', contract_info.purchase_time)) &&
```

Comment:
In SmartTrader there is an option to sell forward starting contract (even before it starts), so I aded this option too